### ISSUE TITLE
Mouf Install on Mac OS X with code versioned thanks to git: The process has been signaled with signal "5".

### ISSUE THREAD

**Issue by vaidiep** [thecodingmachine/mouf#55]
When I install Mouf on Mac OS X 10.9 with my own code versioned thanks to git, I get this error The process has been signaled with signal "5".

My version of git is: git version 1.8.3.4 (Apple Git-47)

**Comment by vaidiep** [comment]
The problem is linked to the compilation of git thanks to the Xcode Command line tools (also used by Macport)

My first solution was to remove my .git folder (and so to not use git for versioning my own code).
But this solution is not possible because I have to use git on this project.

The solution chosen is to change the code of composer and so to get add a custom composer.phar for Mouf.

In the source code of composer, we add this hack in the file Process.php in line 323:

```
if ($this->processInformation['termsig'] != 5 ) {
     //Ignore SIGTRAP
     throw new RuntimeException(sprintf('The process has been signaled with signal "%s".', $this->processInformation['termsig']));
}
```

Then, we compile our own Composer to get a custom composer.phar that we put in the Mouf package.

This solution is temporary. We hope that the next compiled git version on Mac OS X will work properly and then that we will not have to keep this hack so that we could use the default composer.phar provided by Composer as before.

We will continue to check this issue.

**Comment by palypster** [comment]
If it is not about git but about svn, then this could be helpfull:

"This problem should only show itself if you have a PHP script that is calls system("svn") .
It happens because OS X local svn requires the local libsasl installed, but zend's PHP uses its own libsasl build. However, svn and PHP don't share the same address space, just the same environment, so all we need is to give each one the environment it wants."

(from http://forums.zend.com/viewtopic.php?f=8&t=107973#p202868)

Basically what helped me was to add this line:

```
putenv("DYLD_LIBRARY_PATH=''");
```

in the top of the Process.php file of the composer.
